src/hdmint/tracker.cpp

```diff
@@ -372,8 +372,9 @@ bool CHDMintTracker::UpdateStatusInternal(const std::set<uint256>& setMempool, C
     uint256 hashPubcoin = sigma::GetPubCoinValueHash(mint.pubCoinValue);
     //! Check whether this mint has been spent and is considered 'pending' or 'confirmed'
     // If there is not a record of the block height, then look it up and assign it
-    uint256 txidMint;
-    bool isMintInChain = ZerocoinGetSigmaMintTxHash(txidMint, mint.pubCoinValue);
+    COutPoint outPoint;
+    bool isMintInChain = GetOutPoint(outPoint, mint.pubCoinValue);
```

Comment:
We already know the denomination for this so call the pubcoin version will be better to remove trying every denominations.